mxcube3/ho_mediators/beamline_setup.py

```diff
@@ -728,7 +728,7 @@ def dict_repr(self):
 class DetectorDistanceHOMediator(HOMediatorBase):
     def __init__(self, ho, name=''):
         super(DetectorDistanceHOMediator, self).__init__(ho, name)
-        ho.connect("positionChanged", self.value_change)
+        #ho.connect("positionChanged", self.value_change)
```

Comment:
Ill make a proposal for the resolution and detector distance, I think that will be in another PR if you dont mind.